src/fosslight_source/cli.py

```
@@ -409,14 +495,116 @@ def merge_results(
     return scancode_result
 
 
+def _normalize_merge_text(value: str) -> str:
+    return value.strip() if value else ""
+
+
+def _get_merge_licenses(scan_item: SourceItem) -> tuple:
+    return tuple(sorted([license.strip() for license in scan_item.licenses if license and license.strip()]))
```

Ruff (0.15.17)

[error] Variable `license` is shadowing a Python builtin

(A001)